sounds/views.py

```diff
@@ -209,6 +209,8 @@ def sound(request, username, sound_id):
     try:
         sound = Sound.objects.select_related("license", "user", "user__profile", "pack").get(id=sound_id)
         if sound.user.username.lower() != username.lower():
```

Comment:
This check is done 8 other times in this file for various URLs that contain both the sound id and username. We should do the same old username check in those places (preferably in some helper function so that we don't duplicate code)